nuxeo-platform-video-core/src/main/java/org/nuxeo/ecm/platform/video/tools/service/VideoToolsServiceImpl.java

```diff
@@ -139,46 +115,45 @@ public Blob watermark(Blob video, Blob picture, String x, String y) {
     private BlobHolder execute(String toolName, BlobHolder blobHolder, Map<String, Object> parameters) {
         BlobHolder result = null;
 
-        if (isToolAvailable(toolName)) {
-            Class toolClass = videoTools.get(toolName);
-            try {
-
-                VideoTool tool = (VideoTool) toolClass.newInstance();
-                CmdParameters cmdParams = tool.setupParameters(blobHolder, parameters);
-                String commandLineName = tool.COMMANDLINE_NAME;
-
-                CommandLineExecutorService cles = Framework.getService(CommandLineExecutorService.class);
-                ExecResult clResult = cles.execCommand(commandLineName, cmdParams);
-
-                // Get the result, and first, handle errors.
-                if (clResult.getError() != null) {
-                    throw new NuxeoException("Failed to execute the command <" + commandLineName + ">",
-                            clResult.getError());
-                }
-
-                if (!clResult.isSuccessful()) {
-                    throw new NuxeoException("Failed to execute the command <" + commandLineName + ">. Final command [ "
-                            + clResult.getCommandLine() + " ] returned with error " + clResult.getReturnCode());
-                }
-
-                result = tool.buildResult(blobHolder, cmdParams);
+        if (!isToolAvailable(toolName)) {
+            return result;
+        }
+        Class toolClass = videoTools.get(toolName);
```

Comment:
There's really no need to declare this local var here, especially outside the try/catch